library/std/src/panic.rs

```diff
@@ -304,6 +316,10 @@ pub fn get_backtrace_style() -> Option<BacktraceStyle> {
                 BacktraceStyle::Off
             } else if &x == "full" {
                 BacktraceStyle::Full
+            } else if &x == "snippet,full" || &x == "full,snippet" {
+                BacktraceStyle::FullSnippet
+            } else if x.as_os_str().to_str().map_or(false, |s| s.contains(&"snippet")) {
```

Comment:
This seems very permissive, e.g. it will also fire on `RUST_BACKTRACE=nosnippets`

Reply:
I can look for `,snippet` and `snippet,` so that anything that works today for the short backtrace (like "RUST_BACKTRACE=banana") will do what was expected when adding `,snippet`.

Reply:
Did the above